pages.tr/common/base32.md

```diff
@@ -0,0 +1,19 @@
+# base32
+
+> Bir dosya veya standart veriyi Base32 formatında şifrele veya yalın veri çıktısı olarak deşifre et.
+
+- Bir dosyayı şifrele:
```

Comment:
I think you're using the verb `encrypt` (`şifrele`) but the correct verbs are encode and decode.

Reply:
Yeah, maybe.  Google translates `encode` to `kodlamak` 🤔

Reply:
Well, Turkish isn't just a word-to-word mapping of English.  There's likely lots of other, subtle differences.  Let's leave it as is 👍🏻